### PR TITLE
Fix path from XSpec to XSLT

### PR DESCRIPTION
**Pull request by galtm**
# Committer Notes

When running this XSpec file, I got an error because the XSLT file was not found. Probably, some files were moved at some point and the `stylesheet` attribute didn't get updated. With this change, the test now passes.

I also fixed a minor grammatical issue in a scenario label.

### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/usnistgov/metaschema-xslt/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/metaschema-xslt/pulls) for the same update/change?
- [x] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [ ] Do all automated CI/CD checks pass?

### Changes to Core Features:
Not applicable.

<s>- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?</s>
<s>- [ ] Have you written new tests for your core changes, as applicable?</s>
<s>- [ ] Have you included examples of how to use your new feature(s)?</s>
<s>- [ ] Have you updated all [website](https://pages.nist.gov/metaschema) and readme documentation affected by the changes you made? Changes to the website can be made in the website/content directory of your branch.</s>

### CODE REVIEW

**Review by aj-stein-nist — APPROVED** [review]

File: src/testing/issue_235_regression.xspec [hunk 1]
```diff
@@ -2,9 +2,9 @@
 <x:description 
     xmlns:x="http://www.jenitennison.com/xslt/xspec"
     xmlns:mv="http://g"
```
Comment (galtm) [inline src/testing/issue_235_regression.xspec:4]: Hi, @aj-stein-nist . Thanks for the quick code review. I just noticed this namespace declaration on line 4 that isn't being used. Do you want me to push another commit to remove it? It's not causing any harm; merely extraneous.

**Review by wendellpiez — APPROVED** [review]
So awesome, thank you.

File: src/testing/issue_235_regression.xspec (hunk 1 above)
Comment (wendellpiez) [reply src/testing/issue_235_regression.xspec:4]: Sorry @galtm I merged without even looking at that -- we can fix when we come back --

Comment (galtm) [reply src/testing/issue_235_regression.xspec:4]: No worries. It is not a big deal either way. I will leave it alone. Thanks, @wendellpiez .
